mltsp/build_model.py

```diff
@@ -121,6 +122,24 @@ def clean_up_data_dict(data_dict):
         del data_dict['classes'][indices_for_deletion[0]]
     print(len(data_dict['features']))
     print("\n\n")
+    return
+
+
+def fit_model(data_dict):
```

Comment:
Is there any way to reduce the redundancy between `build_model.fit_model`, `build_model. create_and_pickle_model `, and `celery_task_tools.create_and_pickle_model`? Maybe I'm missing some subtlety but it seems like the same fitting code is repeated in each.